soc/ti/k3/j721e/pinctrl_soc.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 Enphase Energy
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_ARM64_TI_K3_PINCTRL_SOC_H_
```

Comment:
Wrong define here.

Reply:
This file is an exact copy of the one in the `soc/ti/k3/am64` directory. The pinctrl should be common for all K3 SoCs, maybe the better solution is to move this file up to the common area. I'll add a patch for that.

Reply:
Ah, sorry. I read `AM64` instead of `ARM64`. This is obviously fine.

Reply:
Yeah, I miss-read that name all the time, but this was still wrong since this file is no longer in the `ARM64` directory either. I've fixed up the definition anyway.